src/animations/cssTransitions.ts

```diff
@@ -0,0 +1,83 @@
+let browserSpecificTransitionEndEventName = '';
+let browserSpecificAnimationEndEventName = '';
+
+interface VNodeProperties {
+	enterAnimationActive?: string;
+	exitAnimationActive?: string;
+}
+
+function determineBrowserStyleNames(element: HTMLElement) {
+	if ('WebkitTransition' in element.style) {
+		browserSpecificTransitionEndEventName = 'webkitTransitionEnd';
+		browserSpecificAnimationEndEventName = 'webkitAnimationEnd';
+	}
+	else if ('transition' in element.style) {
```

Comment:
any reason to not make this be `else if ('transition' in element.style || 'MozTransition' in element.style)` given that the body of each of these if blocks is the same? If the reason is to eventually remove the mozilla specific version, then perhaps at least the if ('transition' in block should come first since that's the standard, or does WebKit evaluate to true for both but need the webkit version?